packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -114,7 +142,11 @@ export default class LinkCore extends React.Component<Props> {
 const styles: Record<string, any> = {};
 
 const iconStyles = StyleSheet.create({
-    icon: {
+    startIcon: {
+        marginRight: Spacing.xxxSmall_4,
+    },
```

Comment:
I think the icon is sitting a bit low within the line height, so it doesn't align with the text.

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/13231763/225746037-5c1e6f77-5216-425c-9d91-5760cd497c1e.png">

Reply:
This worked! I wrapped the link contents in `View` and styled with `display: inline-flex`, `flexDirection: row`, and `alignItems: center`. The flex direction seemed to do the trick. It was set to `column` in my dev tools.

Reply:
<img width="942" alt="Screenshot 2023-03-21 at 1 06 16 PM" src="https://user-images.githubusercontent.com/100858764/226687039-c6ab8919-3ae9-4e17-9ef7-b18f15152922.png">

Reply:
SO I ended up changing this because I noticed when the viewport shrank, the icons got all wonky and didn't stay on the same row as the label:

<img width="573" alt="Screenshot 2023-03-27 at 3 51 49 PM" src="https://user-images.githubusercontent.com/100858764/228055351-5ca9ccd3-9207-4333-88cf-abc12cd289a6.png">

I tried a bunch of different things still wrapping `linkContent` in a `View` but ultimately got rid of `View` and got the closest by just adding `Spacing.xxxxSmall_2` to the top & bottom margin.

Here's what it looks like now:

<img width="566" alt="Screenshot 2023-03-27 at 4 13 50 PM" src="https://user-images.githubusercontent.com/100858764/228055904-dc38c745-5f70-422f-8cd5-10410a719ba5.png">